intel_extension_for_transformers/neural_chat/requirements_hpu.txt

```diff
@@ -6,6 +6,7 @@ fschat==0.2.32
 git+https://github.com/EleutherAI/lm-evaluation-harness.git@cc9778fbe4fa1a709be2abed9deb6180fd40e7e2
 intel_extension_for_pytorch
 neural-compressor
+neural_speed
```

Comment:
why hpu requires neural-speed?

Reply:
When I verified the code, an error occured: <img width="496" alt="image" src="https://github.com/intel/intel-extension-for-transformers/assets/106130696/10e50350-5f9b-4aaa-8ffa-8f4d9532069e">,after `pip install neural-speed`, it runs smootly.
<img width="845" alt="image" src="https://github.com/intel/intel-extension-for-transformers/assets/106130696/85a4ca96-309c-4c5d-ab10-930b4d9689b5">

Reply:
neural speed has nothing related with hpu. so please root cause the issue.

Reply:
Solved. neural speed is no longer imported on hpu now.